cmd/cariddi/main.go

```diff
@@ -90,6 +90,12 @@ func main() {
 		StoreResp:     flags.StoreResp,
 	}
 
+	config.OutputDir = output.CariddiOutputFolder
+	if flags.StoredRespDir != "" {
```

Comment:
I think we should perform some tests. What if config.StoreResp is not specified?

Reply:
With the two tags `-sr` and `-srd`, there are four possible scenarios:

1. **Both `-sr` and `-srd` are not set:**
   - No HTTP response will be saved.
   - `OutputDir` will be set to its default value, and any other data that needs to be saved will be stored in this directory.

2. **Only `-sr` is set:**
   - The HTTP response, along with any other data, will be saved in the `OutputDir`, which will be the default directory.

3. **Only `-srd` is set:**
   - Everything, including the HTTP response, will be saved to the `OutputDir` provided with the `-srd` flag.

4. **Both `-sr` and `-srd` are set:**
   - Everything will be saved to the `OutputDir` provided with the `-srd` flag.

In the config struct, I have used the `StoreResp` boolean as the key indicator of whether to store the response. Therefore, if the user provides only the `-srd` flag, we will set `StoreResp` to `true`.

Reply:
As far as I can understand point 3 and 4 are the same right?

Reply:
Yes, just in point 3 we will set the StoreResp value to true